csharp-selenium-webdriver-sample/WebDriverFactory.cs

```diff
@@ -0,0 +1,37 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using OpenQA.Selenium;
+using OpenQA.Selenium.Chrome;
+using OpenQA.Selenium.Firefox;
+using System;
+
+namespace CSharpSeleniumWebdriverSample
+{
+    public enum BrowserType
+    {
+        Chrome,
+        Firefox,
+    }
+
+    public class WebDriverFactory
+    {
+        public static IWebDriver GetWebDriver(BrowserType browser)
+        {   
+            switch (browser)
+            {
+                case BrowserType.Chrome:
+                    // Check if Chrome web driver eniroment varibale already defined so that Chrome version and the webdriver are the same, other wise use Selenium.WebDriver.ChromeDriver to indtall it
```

Comment:
Some misspellings in this comment; I'm okay with omitting it until the separate PR where we add the more verbose comments throughout